app/utils/company_list.py

```diff
@@ -57,6 +56,34 @@ def get(cls):
             LOG.error(err)
         return cls(company_list)
 
+    @classmethod
+    async def get_async(cls):
```

Comment:
Added a class method for async operations. The existing sync method has not been removed as it is used in batches.